.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: emacs-lisp
+env:
+  - EVM_EMACS=emacs-24.3-travis
+  - EVM_EMACS=emacs-24.4-travis
+  - EVM_EMACS=emacs-24.5-travis
```

Comment:
I think having 3 e24 environments is an overkill. How about saving some trees and leaving only 24.3 there?

Reply:
Good point, if it works on 24.3 chances are it works on 24.4 and 24.5.